data/en/lms_2.json

```diff
@@ -687,15 +687,311 @@
                         }],
                         "routing_rules": [{
                             "goto": {
-                                "block": "country-of-birth"
+                                "block": "country-of-birth-scotland",
```

Comment:
Would it be more concise / appropriate to use `skip_conditions` for this scenario.
As it should remove the amount of `routing_rules`

Reply:
Yeah, makes sense

Reply:
Fixed now. Looks much better.